terraform/revisions.tf

```diff
@@ -0,0 +1,22 @@
+locals {
+  postgresql_revisions = {
+    amd64 = 468,
```

Comment:
@taurus-forever : We'll probably need update bundle workflow here, correct? If so, I'll create a ticket for it.

Reply:
Yes, now we have too. Please create a followup ticket.

Reply:
https://warthogs.atlassian.net/browse/DPE-5673